domain-server/src/DomainServer.h

```diff
@@ -234,8 +236,10 @@ private slots:
     std::vector<QString> _replicatedUsernames;
 
     DomainGatekeeper _gatekeeper;
+    DomainServerExporter _exporter;
 
     HTTPManager _httpManager;
+    HTTPManager *_httpExporterManager { nullptr };
```

Comment:
Pointer operator should be attached to the type to keep it consistent in the codebase
```suggestion
    HTTPManager* _httpExporterManager { nullptr };
```